types/esri-leaflet/index.d.ts

```diff
@@ -596,6 +611,148 @@ declare module 'leaflet' {
+         * Image Services provide access to raster data through a web service.
+         */
+
+        class ImageMapLayer extends RasterLayer {
+            constructor(options: ImageMapLayerOptions);
+
+            /**
+             * Uses the provided function to create a popup that will identify pixel value(s) whenever the map is clicked.
+             * Your function will be passed an object with a `pixel` property that is a GeoJSON Point with the pixel
+             * value(s) at the clicked location and should return the appropriate HTML. If you do not want to open the
+             * popup when there are no results, return false.
+             *
+             * NOTE: by default, if the layer has a mosaic rule applied, then the same rule will be applied to the
+             * identify request. Conversely, if the layer has a rendering rule applied, that rule is NOT applied to the
+             * layer so that the raw pixel value can be returned. If you need specific control over how these rules
+             * (and/or other identify parameters) are passed to the identify service, use L.esri.IdentifyImage.
+             */
+            bindPopup(
+                fn: LayerCallbackHandler | ((layer: Layer) => Content) | string | HTMLElement | Popup,
+                popupOptions?: PopupOptions,
+            ): this;
+
+            /**
+             * Removes a popup previously bound with bindPopup.
+             */
+            unbindPopup(): this;
+
+            /**
+             * Returns the current band value(s).
+             */
+            getBandIds(): string;
+
+            /**
+             * Specify a single band to export, or you can change the band combination (red, green, blue) by
+             * specifying the band number.
+             */
+            setBandIds(bandIds: string | string[]): this;
+
+            /**
+             * Returns the current no data value.
+             */
+            getNoData(): string;
+
+            /**
+             * Specify a single value, or an array of values to treat as no data. No data will values will be rendered transparent.
+             * The optional `noDataInterpretation` can be either `esriNoDataMatchAny` | `esriNoDataMatchAll`. The
+             * default is `esriNoDataMatchAny` when `noData` is a number, and `esriNoDataMatchAll` when noData is an array.
+             * See Image Service Export Image documentation for more details
+             */
+            setNoData(noData: number | number[], noDataInterpretation?: string): this;
+
+            /**
+             * Returns the current no data interpretation value.
+             */
+            getNoDataInterpretation(): string;
+
+            /**
+             * Returns the current pixel type.
+             */
+            getPixelType(): string;
+
+            /**
+             * 	The pixel type, also known as data type, pertains to the type of values stored in the raster, such as
```

Comment:
<!--
  1 failure:  The root .editorc...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: types/esri-leaflet/index.d.ts;
  Line: 714;
-->
- :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.